Example/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "dependencies": {
     "react": "~15.3.0",
-    "react-native": "^0.34.0",
+    "react-native": "^0.41.2",
```

Comment:
My Bad - I should have removed the version change from the pull request.  Will do so.